README.md

```diff
@@ -1,188 +1,71 @@
 # JWTDecode.Android
 
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/JWTDecode.Android.svg?style=flat-square)](https://circleci.com/gh/auth0/JWTDecode.Android/tree/master)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0.android/jwtdecode.svg?style=flat-square)](https://search.maven.org/artifact/com.auth0.android/jwtdecode)
 [![codecov](https://codecov.io/gh/auth0/JWTDecode.android/branch/master/graph/badge.svg)](https://codecov.io/gh/auth0/JWTDecode.android)
-[![Download](https://api.bintray.com/packages/auth0/android/jwtdecode/images/download.svg)](https://bintray.com/auth0/android/jwtdecode/_latestVersion)
+[![javadoc](https://javadoc.io/badge2/com.auth0.android/jwtdecode/javadoc.svg)](https://javadoc.io/doc/com.auth0.android/jwtdecode)
 
-Java library with focus on Android that provides Json Web Token (JWT) decoding.
+📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • ⏭️ [Next Steps](#next-steps) • 💬 [Feedback](#feedback)
 
-## Install
+## Documentation
+
+- [Docs Site](https://auth0.github.io/react-native-auth0/)
+- [API Reference](https://javadoc.io/doc/com.auth0.android/jwtdecode/latest/index.html)
+- [Examples](https://github.com/auth0/JWTDecode.Android/blob/master/EXAMPLES.md)
+
+## Getting Started
+
+### Installation
 The library is be available both in Maven Central and JCenter. To start using it add this line to your `build.gradle` dependencies file:
 
 ```groovy
 implementation 'com.auth0.android:jwtdecode:2.0.1'
 ```
 
-## Usage
+## Next Steps
```

Comment:
I think this could potentially be folded under `Getting Started` as `Usage` or similar to keep the sections consistent between the repos, maybe like how Fred did in the [JS jwt-decode readme](https://github.com/auth0/jwt-decode#getting-started)